Update SNEWS Mission Page (#3729)
nasa-gcn/gcn.nasa.gov: app/routes/missions.snews/route.mdx

| ------- | ---------------------------- | ------- |
| `SNEWS` | REAL or TEST supernova alert | Minutes |

<WithFeature SNEWS_2>

**JSON-Serialized SNEWS2 Notices in GCN Kafka:**

> SNEWS2 distributes JSON-serialized notices for neutrino coincidence alerts generated by the SuperNova Early Warning System (SNEWS 2.0) on the GCN Kafka.

Reviewer: ```suggestion
SNEWS2 distributes notices for neutrino coincidence alerts generated by the SuperNova Early Warning System (SNEWS 2.0) in the JSON format using GCN Kafka.
```

> SNEWS2 distributes JSON-serialized notices for neutrino coincidence alerts generated by the SuperNova Early Warning System (SNEWS 2.0) coincidence network on the GCN Kafka topic `gcn.notices.snews2.coincidence`.

Author: ```suggestion
SNEWS2 distributes JSON-serialized notices for neutrino coincidence alerts generated by the SuperNova Early Warning System (SNEWS 2.0) on the GCN Kafka topic `gcn.notices.snews2.coincidence`.
```
